### ISSUE TITLE
Create Rectangle + Enter - Unable to enter text

### ISSUE THREAD

**Issue by ruturajv-media-net** [excalidraw/excalidraw#11358]
Earlier, when creating a rectangle (`R` or `2`)  and then pressing enter would enable the rectangle's text field which was easy accessibility. But now it doesn't work. Now the user has to click the rectangle again ( with arrow tool after changing it ) and then hit Enter.

STR:

1. Create a Rectangle
2. Hit `Enter` on keyboard

Expected:

The rectangle's text box should be activated / focussed

Observed:

The rectangle loses focus, so the `Enter` key doesn't target the rectangle.

**Comment by ruturajv-media-net** [comment]
sorry, **Keep Selected Tool active** (`Q`) during drawing was active.
